app/src/main/java/com/elmeradrianv/shesafe/database/TypeOfCrime.java

```diff
@@ -0,0 +1,24 @@
+package com.elmeradrianv.shesafe.database;
+
+import com.parse.ParseClassName;
+import com.parse.ParseObject;
+
+@ParseClassName("TypeOfCrime")
+public class TypeOfCrime extends ParseObject {
+    public static final String TAG_KEY = "tag";
+    public static final String LEVEL_OF_RISK_KEY = "levelOfRisk";
+    public static final int LOW_RISK=1;
+    public static final int MEDIUM_LOW_RISK=2;
+    public static final int MEDIUM_RISK=3;
+    public static final int MEDIUM_HIGH_RISK=4;
+    public static final int HIGH_RISK=5;
```

Comment:
This seems like a good use of an enum, indicate that these are all related to each other.

    public enum RiskLevel{
        Low(0),
        MediumLow(1),...;

        private final int value;

        RiskLevel(final int newValue) {
            value = newValue;
        }

        public int getValue() { return value; }
    }